contracts/Tokens/VTokens/interfaces/IVBep20.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+import { IVToken } from "./IVToken.sol";
+
+interface IVBep20 is IVToken {
```

Comment:
Missing natspec for all functions in the interface?

Reply:
c974cf206a7fc31c5b03ee5f46f4d94228c62d12